packages/snap-client/src/types.ts

```diff
@@ -107,6 +107,21 @@ export type RecommendRequestModel = {
 	batched?: boolean;
 	limits?: number | number[];
 	order?: number;
+	filters?: RecommendationRequestFilterModel[];
+};
+
+export type GetRecommendRequestModel = Omit<RecommendRequestModel, 'filters'> & {
+	[filter: `filter.${string}`]: (string | number)[];
+};
+
+export type PostRecommendationRequestFiltersModel = Omit<RecommendRequestModel, 'filters'> & {
```

Comment:
Isn't this just the `PostRecommendationRequestModel`, not sure why it is named differently (see `GetRecommendRequestModel` above).